.gitignore

```diff
@@ -1,3 +1,4 @@
 /node_modules/
 /build/
 *.local.js
+.idea
```

Comment:
fwiw this doesn't hurt to add, but it's the sort of thing that should be in your global gitignore instead of adding intellij pollution to random projects